charts/gitops-runtime/Chart.yaml

```diff
@@ -47,7 +47,7 @@ dependencies:
   condition: tunnel-client.enabled
 - name: codefresh-gitops-operator
   repository: oci://quay.io/codefresh/charts
-  version: 1.0.21
+  version: 1.0.22
```

Comment:
on annotations.artifacthub.io/changes dont we specifying the version anymore? should I just add another description?

@ilia-medvedev-codefresh